app/src/components/ui/button.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 import { cn } from '@/lib/utils';
 
 const buttonVariants = cva(
-    'inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50',
+    'inline-flex items-center justify-center whitespace-nowrap rounded-md text-smallPlus transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50',
```

Comment:
@drfarrellfor reference, the convention for `smallPlus` should be `small-plus` which mans we should update the token names. 

```
 "small-plus": [
        '0.8125rem',
        {
            lineHeight: '1.3rem',
            fontWeight: '500',
            letterSpacing: '0.00rem',
        },
    ],
```